### PR TITLE
Prevent from scrolling past the end of the content

### PR DESCRIPTION
**Pull request by georgijd**
Fixes issue #31

### CODE REVIEW

File: wuzz.go [hunk 1]
```diff
@@ -781,6 +781,10 @@ func scrollView(v *gocui.View, dy int) error {
 	if oy+dy < 0 {
 		dy = -oy
 	}
+	_, err := v.Line(dy)
+	if dy > 0 && err != nil {
```
Comment (antham) [inline wuzz.go]: You could write 
```go
if _, err := v.Line(dy); dy > 0 && err != nil {
    dy = 0
}
```

**Comment by georgijd** [comment]
Done :)

**Comment by asciimoo** [comment]
That was fast, thanks =)

**Comment by Benaiah** [comment]
Woah, I've never been in a GitHub conversation that moved fast enough that I saw the thread update live. Thanks @georgijd!

**Pull request merged**